homeassistant/components/hue/v2/group.py

```diff
@@ -230,6 +230,12 @@ def _update_values(self) -> None:
         all_lights = self.controller.get_lights(self.resource.id)
         lights_in_colortemp_mode = 0
         lights_in_dynamic_mode = 0
+        # accumulate colour values
```

Comment:
nit: "colour" and "color" are both used in comments. HA generally uses US English, so just "color" is preferred.